### PR TITLE
feat(i18n): add ukranian translations

### PR DESCRIPTION
**Pull request by GlugovGrGlib**
### SUMMARY
Adding initial version of the Ukrainian translations for the Superset. To translate phrases I used Google Translate, and after this manually checked most of the translations.

### AFTER SCREENSHOTS
Screenshot 2023-09-17 at 01-16-05 USA Births Names
Screenshot 2023-09-17 at 01-16-34 Superset
Screenshot 2023-09-17 at 01-17-25 Superset
Screenshot 2023-09-17 at 01 17 10

### TESTING INSTRUCTIONS
- activate uk (Ukrainian) language in superset_config.py
- open the dashboard, navigate through the system

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/translations/uk/LC_MESSAGES/messages.json [hunk 1]
```diff
@@ -0,0 +1,6401 @@
...
+      ],
+      ".": ["."],
+      "0 Selected": ["0 Вибрано"],
+      "1 calendar day frequency": ["1 Календарний день частота"],
+      "1 day": ["1 день"],
+      "1 day ago": ["1 день тому"],
+      "1 hour": ["1 година"],
+      "1 hourly frequency": ["1 погодинна частота"],
+      "1 minute": ["1 хвилина"],
+      "1 minutely frequency": ["1 Частотна частота"],
```
Comment (mlisivka) [inline superset/translations/uk/LC_MESSAGES/messages.json:127]:
```suggestion
      "1 minutely frequency": ["1 хвилинна частота"],
```

File: superset/translations/uk/LC_MESSAGES/messages.json [hunk 2]
```diff
@@ -0,0 +1,6401 @@
...
+      "15 minute": ["15 хвилин"],
+      "156 weeks": ["156 тижнів"],
+      "156 weeks ago": ["156 тижнів тому"],
+      "1AS": ["1AS"],
+      "1D": ["1D"],
+      "1H": ["1H"],
+      "1M": ["1M"],
+      "1T": ["1T"],
+      "2 years": ["2 роки"],
+      "2 years ago": ["2 роки назад"],
```
Comment (mlisivka) [inline superset/translations/uk/LC_MESSAGES/messages.json:154]:
```suggestion
      "2 years ago": ["2 роки тому"],
```

File: superset/translations/uk/LC_MESSAGES/messages.json [hunk 3]
```diff
@@ -0,0 +1,6401 @@
...
+      "A dictionary with column names and their data types if you need to change the defaults. Example: {\"user_id\":\"integer\"}": [
+        "Словник із назвами стовпців та їх типами даних, якщо вам потрібно змінити за замовчуванням. Приклад: {\"user_id\": \"Integer\"}"
+      ],
+      "A full URL pointing to the location of the built plugin (could be hosted on a CDN for example)": [
+        "Повна URL -адреса, що вказує на розташування вбудованого плагіна (наприклад, може бути розміщена на CDN)"
+      ],
+      "A handlebars template that is applied to the data": [
+        "Шаблон ручки, який застосовується до даних"
+      ],
+      "A human-friendly name": ["Ім'я зручної для людини"],
```
Comment (mlisivka) [inline superset/translations/uk/LC_MESSAGES/messages.json:219]: I think it's better to have
"Ім'я, що зручне для людини"
Or
"Зручне для людини ім'я"

File: superset/translations/uk/LC_MESSAGES/messages.json [hunk 4]
```diff
@@ -0,0 +1,6401 @@
...
+      "A valid color scheme is required": ["Потрібна дійсна кольорова гама"],
+      "APPLY": ["Застосовувати"],
+      "APR": ["Квітня"],
+      "AQE": ["AQE"],
+      "AUG": ["Серпень"],
+      "AXIS TITLE MARGIN": ["ЗАВДАННЯ ВІСІВ"],
+      "AXIS TITLE POSITION": ["Позиція заголовка вісь"],
+      "About": ["Про"],
+      "Access": ["Доступ"],
+      "Access requests": ["Запити на доступ до доступу"],
```
Comment (mlisivka) [inline superset/translations/uk/LC_MESSAGES/messages.json:284]:
```suggestion
      "Access requests": ["Запити доступу"],
```

File: superset/translations/uk/LC_MESSAGES/messages.json [hunk 5]
```diff
@@ -0,0 +1,6401 @@
...
+      "Add item": ["Додати елемент"],
+      "Add metric": ["Додати показник"],
+      "Add metrics to dataset in \"Edit datasource\" modal": [
+        "Додайте показники до набору даних у модал \"Редагувати DataSource\""
+      ],
+      "Add new color formatter": ["Додайте новий кольоровий форматер"],
+      "Add new formatter": ["Додати новий форматер"],
+      "Add notification method": ["Додайте метод сповіщення"],
+      "Add required control values to preview chart": [
+        "Додайте необхідні контрольні значення до діаграми попереднього перегляду"
```
Comment (mlisivka) [inline superset/translations/uk/LC_MESSAGES/messages.json:352]:
```suggestion
        "Додайте необхідні контрольні значення для попереднього перегляду діаграми"
```

**Review by mlisivka — COMMENTED** [review]
I just looked through some of the translations.

I think the word "дашборд" is good to change "панель" or "табло" or a bit different "упорядник"

Also, I found that you use 2 different words for "Chart": "Діаграма" and "Графік". I think "Діаграма" is more general and fits better

File: superset/translations/uk/LC_MESSAGES/messages.json (hunk 1 above)
Comment (GlugovGrGlib) [reply superset/translations/uk/LC_MESSAGES/messages.json]: Thanks, updated

File: superset/translations/uk/LC_MESSAGES/messages.json (hunk 2 above)
Comment (GlugovGrGlib) [reply superset/translations/uk/LC_MESSAGES/messages.json]: Updated

File: superset/translations/uk/LC_MESSAGES/messages.json (hunk 3 above)
Comment (GlugovGrGlib) [reply superset/translations/uk/LC_MESSAGES/messages.json]: Changed

File: superset/translations/uk/LC_MESSAGES/messages.json (hunk 4 above)
Comment (GlugovGrGlib) [reply superset/translations/uk/LC_MESSAGES/messages.json]: Fixed this

File: superset/translations/uk/LC_MESSAGES/messages.json (hunk 5 above)
Comment (GlugovGrGlib) [reply superset/translations/uk/LC_MESSAGES/messages.json]: Thanks, updated

**Comment by GlugovGrGlib** [comment]
> I just looked through some of the translations.
> 
> I think the word "дашборд" is good to change "панель" or "табло" or a bit different "упорядник"
> 
> Also, I found that you use 2 different words for "Chart": "Діаграма" and "Графік". I think "Діаграма" is more general and fits better

Thank you, I fixed Chart translations. As per “Dashboard” I am working on translating it to “Інформаційна Панель” throughout the pages.

**Comment by mlisivka** [comment]
@michael-s-molina OK, but it will take some time for me, probably weeks. Ok?

**Review by rusackas — APPROVED** [review]
LGTM!

**Comment by rusackas** [comment]
It seems this PR was getting stuck, and I didn't want to lose all this value in a conflict or something. The net benefit seems very high, even if there are a few things to correct in follow-up PRs. Please do open as many PRs as you'd like to make corrections and improve the situation.

**Pull request merged**